docs/docs/rest-api/public/api/v2/types/queue.raml

```diff
@@ -66,6 +66,20 @@ types:
         type: boolean
         description: true, if this run spec is backed off. Otherwise false.
 
+  DelayResult:
+    properties:
+      queueDelay?:
+        type: QueueDelay
+        description: If the app is currently queued, show queue delay statistics.
+      currentDelaySeconds:
```

Comment:
I like the deadline timestamp a little better than deltas. Or we add the `referenceTimestamp` WDYT?

Reply:
ISO 8601 should include the timezone. We can also advertise that it is always UTC 🙂.